lib/cdo/create_header.rb

```diff
@@ -33,11 +49,15 @@ def self.get_create_dropdown_contents(options)
     applab_gamelab = %w(applab gamelab)
 
     entries = options[:limit_project_types] == "true" ?
-      everyone_entries + ["minecraft"] :
+      everyone_entries + ["minecraft_designer"] :
       everyone_entries + applab_gamelab
 
     entries << "dance"
```

Comment:
this is a fair bit harder to follow than:
```ruby
entries = options[:limit_project_types] ?
  %w{spritelab artist minecraft_designer dance} :
  %w{spritelab artist applab gamelab dance}
```

Reply:
oh, good catch. I went with the smallest delta from how it currently is for my first pass, then forgot to go back and clean it up. Thanks!